BUILDING.md

```diff
@@ -172,14 +172,13 @@ Binaries at <https://nodejs.org/download/release/> are produced on:
 | darwin-x64 (and .pkg) | macOS 10.15, Xcode Command Line Tools 11 with -mmacosx-version-min=10.13 |
 | linux-arm64           | CentOS 7 with devtoolset-6 / GCC 6                                       |
 | linux-armv7l          | Cross-compiled on Ubuntu 16.04 x64 with [custom GCC toolchain](https://github.com/rvagg/rpi-newer-crosstools)   |
```

Comment:
@sam-github @rvagg where are we re. the arm builds?

Reply:
Last night's daily built and passed on gcc 8: https://ci.nodejs.org/job/node-test-commit-arm/30719/nodes=centos7-arm64-gcc8/consoleFull  I'm not familiar enough with arm architecture variants to comment much past that.

https://ci.nodejs.org/job/node-cross-compile/28583/nodes=cross-compiler-armv7-gcc-6/ is the fanned build, it uses ubuntu 16.04 and gcc 6. 

The difference between node-test-commit-arm and node-test-commit-arm-fanned is not clear from the names -- it looks like the "fanned" is an implementation optimization, but the architectures are different? Maybe fanned is armv7l, and non-fanned is arm64?

Reply:
Been thinking a bit more on this. I don't think we should switch to gcc8 for the armv7 binaries unless there's a reason V8 will force that on us. The only reason we can get away with that on the centos7 builds is because of devtoolset libc magic which we don't get outside of devtoolset.

But we are going to have to deal with the Ubuntu 16.04 cross compiler machines. I think you should change that to 18.04 here but it needs the work to be done - initially just adding 3 new cross compilers to CI (2 ci, 1 release) and maybe migrating the older ones across if we can prove stability of the binaries. I guess this is all on me! What's our timeline with 14?

Reply:
Updated cross compiler machine to Ubuntu 18.04. Current timeline as I understand it is that @BethGriggs was planning a final update/rc build this week after [v13.13.0](https://github.com/nodejs/node/pull/32813) goes out (so that the changelog for 14.0.0 can be generated as a diff in commits to it) with the release planned for next week (21 April 2020).

Reply:
In terms of timeline for v14.x, I'm going to integrate the last set of functional changes today and start finalising the changelog. But, I can easily integrate documentation PR's like these up until ~Monday 20th April.  Release planned to go out 21st April.